django/conf/global_settings.py

```diff
@@ -529,6 +529,7 @@ def gettext_noop(s):
     'django.contrib.auth.hashers.PBKDF2SHA1PasswordHasher',
     'django.contrib.auth.hashers.Argon2PasswordHasher',
     'django.contrib.auth.hashers.BCryptSHA256PasswordHasher',
+    'django.contrib.auth.hashers.ScryptPasswordHasher',
```

Comment:
Does it make sense to add the hasher here? It will not get used anyways *scratches head*

Reply:
I removed the line from the global_settings.py.

Reply:
I think it's worth, re-added.